includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -0,0 +1,115 @@
+	 */
+	public static function register_advanced_fields() {
+		\add_settings_section(
+			'activitypub_advanced_settings',
+			\__( 'Advanced Settings', 'activitypub' ),
+			array( self::class, 'render_advanced_settings_section' ),
+			'activitypub_advanced_settings'
+		);
+
+		add_settings_field(
+			'activitypub_outbox_purge_days',
+			__( 'Outbox Retention Period', 'activitypub' ),
+			array( self::class, 'render_outbox_purge_days_field' ),
+			'activitypub_advanced_settings',
+			'activitypub_advanced_settings',
+			array( 'label_for' => 'activitypub_outbox_purge_days' )
+		);
+
+		\add_settings_field(
+			'activitypub_authorized_fetch',
+			__( 'Authorized Fetch', 'activitypub' ),
+			array( self::class, 'render_authorized_fetch_field' ),
+			'activitypub_advanced_settings',
+			'activitypub_advanced_settings',
+			array( 'label_for' => 'activitypub_authorized_fetch' )
+		);
+	}
+
+	/**
+	 * Render Advanced Settings Section.
+	 */
+	public static function render_advanced_settings_section() {
+		?>
+		<p>
+			<?php
+			$allowed_html = array(
+				'a' => array(
+					'href'   => true,
+					'target' => true,
+				),
+			);
+			echo wp_kses( __( 'Advanced settings allow deep customization but can affect your site&rsquo;s functionality, security, or performance if misconfigured. Only proceed if you fully understand the changes, and always back up your site beforehand. If unsure, consult <a href="https://github.com/Automattic/wordpress-activitypub/tree/trunk/docs" target="_blank">documentation</a> or seek <a href="https://wordpress.org/support/plugin/activitypub/" target="_blank">expert advice</a>.', 'activitypub' ), $allowed_html );
+			?>
+		</p>
+		<?php
+	}
+
+	/**
+	 * Render outbox purge days field.
+	 */
+	public static function render_outbox_purge_days_field() {
+		$value = get_option( 'activitypub_outbox_purge_days', 180 );
+		echo '<input type="number" id="activitypub_outbox_purge_days" name="activitypub_outbox_purge_days" value="' . esc_attr( $value ) . '" class="small-text" min="0" max="365" />';
+		echo '<p class="description">' . wp_kses(
+			sprintf(
+				// translators: 1: Definition of Outbox; 2: Default value (180).
+				__( 'Maximum number of days to keep items in the <abbr title="%1$s">Outbox</abbr>. A lower value might be better for sites with lots of activity to maintain site performance. Default: <code>%2$s</code>', 'activitypub' ),
+				esc_attr__( 'A virtual location on a user&#8217;s profile where all the activities (posts, likes, replies) they publish are stored, acting as a feed that other users can access to see their publicly shared content', 'activitypub' ),
+				esc_html( 180 )
+			),
+			array(
+				'abbr' => array( 'title' => array() ),
+				'code' => array(),
+			)
+		) . '</p>';
+	}
+
+	/**
+	 * Render use Authorized Fetch field.
+	 */
+	public static function render_authorized_fetch_field() {
+		$disabled = '';
+		if ( defined( 'ACTIVITYPUB_AUTHORIZED_FETCH' ) ) {
+			$disabled = 'disabled';
```

Comment:
Instead of disabling it, should we replace it with the same message as we do for actor mode and the interactions setting?